notebooks/llm-agent-langchain/llm-agent-langchain.ipynb

```diff
@@ -84,7 +84,7 @@
   },
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #33.    <code></code>`[\INST]"""
</pre><p>should it be \\INST?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/2094/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

Reply:
<p>It should be [/INST], just fixed</p>